Algorithms/Search/JumpSearcher.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+namespace Algorithms.Search {
+
+    /// <summary>
+    /// The idea: This is one of the public search algorithm.
+    /// It takes 2 argument array of numbers and value of number
+    /// It finds index of this value in given array
+    ///     Time compl is O(root(n))
+    /// Note: the array has to be sorted beforehand.
+    /// </summary>
+    public class JumpSearch {
+        /// <summary>
+        /// Code is really easy we find step size first
+        /// Later we find the block where this element should be.!
+        /// In this block we go search for value.! If found return index
+        /// Otherwise return -1!
+        /// </summary>
+        private int jumpSearch (int[] arr, int val) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'jumpSearch' to match pascal case naming rules, consider using 'JumpSearch'.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=5684457)

Reply:
Will change to it!